### PR TITLE
[RA2 Ch4] Change to specifications

### LINKED ISSUE #1525: [RA2 Ch04] Modify specifications into table with references
As per suggestion in #1186:
https://github.com/cntt-n/CNTT/issues/1186#issuecomment-594448078

### PR DESCRIPTION
**Pull request by tomkivlin**
Fixes #1525

### CODE REVIEW

**Comment by tomkivlin** [comment]
@CsatariGergely @rabi-abdel @peterwoerndle - can I just ask for a quick check on the structure (I have just commented out existing content for now) please?  If happy I will move over more of the existing content into the table format.  PR is in draft state for now - so just comment here for now, thanks.

**Comment by CsatariGergely** [comment]
I think this structure is good. I can help restructuring the networking part.

**Comment by rabi-abdel** [comment]
@tomkivlin the structure looks really good. it covers the entire stack bottom up.

**Comment by pgoyal01** [comment]
@tomkivlin do we need a section of host configuration? The section on host config will include topics such as configuring CPU dedicated resources for Host kernel, CPU isolation, etc.; Memory resource config such as for hugepages; Networking and local storage configs

**Comment by tomkivlin** [comment]
> @tomkivlin do we need a section of host configuration? The section on host config will include topics such as configuring CPU dedicated resources for Host kernel, CPU isolation, etc.; Memory resource config such as for hugepages; Networking and local storage configs

Yes good point @pgoyal01. Let's change 4.2 to just Container Host and cover machine-level and OS-level configuration as I think the two could be quite linked - e.g. BIOS settings/boot loader settings/kernel modules/etc.?

**Comment by tomkivlin** [comment]
@cntt-n/ref-a2 for info, I am proposing we push this change as-is and then use separate PRs to move content from commented areas into the tables.

File: doc/ref_arch/kubernetes/chapters/chapter04.md [hunk 1]
```diff
@@ -5,29 +5,49 @@
 
 ## Table of Contents
 * [4.1 Introduction](#4.1)
-* [4.2 Host OS](#4.2)
+* [4.2 Container Host](#4.2)
```
Comment (xmulligan) [inline doc/ref_arch/kubernetes/chapters/chapter04.md:8]:
```suggestion
* [4.2 Container Host OS](#4.2)
```

Comment (xmulligan) [reply doc/ref_arch/kubernetes/chapters/chapter04.md:8]: To match with naming below

Comment (tomkivlin) [reply doc/ref_arch/kubernetes/chapters/chapter04.md:8]: The naming below is just `Container Host`?  Which line do you mean?

Comment (xmulligan) [reply doc/ref_arch/kubernetes/chapters/chapter04.md:8]: line 23

File: doc/ref_arch/kubernetes/chapters/chapter04.md [hunk 2]
```diff
@@ -5,29 +5,49 @@
...
+* [4.9 Kubernetes workloads](#4.9)
+* [4.10 Supplementary components](#4.10)
 
 <a name="4.1"></a>
 ## 4.1 Introduction
 
-This chapter describes in detail the CNTT Kubernetes Reference Architecture in terms of the functional capabilities and how they relate to the Reference Model requirements, i.e. how the infrastructure profiles are determined, documented and delivered.
+This chapter describes in detail the Kubernetes Reference Architecture in terms of the functional capabilities and how they relate to the Reference Model requirements, i.e. how the infrastructure profiles are determined, documented and delivered.
+
+The specifications defined in this chapter will be detailed with unique identifiers, which will follow the pattern: `ra2.<section>.<index>`, e.g. `ra2.ch.001` for the first requirement in the Container Host OS section.  These specifications will then be used as requirements input for the Kubernetes Reference Implementation and any Vendor or Community Implementations.
```
Comment (tomkivlin) [inline doc/ref_arch/kubernetes/chapters/chapter04.md:23]:
```suggestion
The specifications defined in this chapter will be detailed with unique identifiers, which will follow the pattern: `ra2.<section>.<index>`, e.g. `ra2.ch.001` for the first requirement in the Container Host section.  These specifications will then be used as requirements input for the Kubernetes Reference Implementation and any Vendor or Community Implementations.
```

File: doc/ref_arch/kubernetes/chapters/chapter04.md (hunk 1 above)
Comment (tomkivlin) [reply doc/ref_arch/kubernetes/chapters/chapter04.md:8]: ah thanks - I've corrected that line now - ta

**Pull request merged**